pyvista/core/filters/rectilinear_grid.py

```diff
@@ -0,0 +1,101 @@
+"""Filters module with class to manage filters/algorithms for rectilinear grid datasets."""
+
+import collections
+from typing import Sequence, Union
+
+import numpy as np
+
+from pyvista import _vtk, abstract_class
+from pyvista.core.filters import _get_output, _update_alg
+
+
+@abstract_class
+class RectilinearGridFilters:
+    """An internal class to manage filters/algorithms for rectilinear grid datasets."""
+
+    def to_tetrahedra(
+        self,
+        tetra_per_cell: int = 5,
+        mixed: Union[Sequence[int], bool] = False,
+        pass_cell_ids: bool = False,
+        progress_bar: bool = False,
+    ):
+        """Create a Tetrahedral mesh from a RectilinearGrid.
+
+        Parameters
+        ----------
+        tetra_per_cell : int, default: 5
+            The number of tetrahedrals to divide each cell in the into. Can be
+            either ``5``, ``6``, or ``12``. If ``mixed=True``, this value is
+            overridden.
+
+        mixed : str, bool, sequence, default: False
+            When set, subdivides some cells into 5 and some cells into 12. Set
+            to ``True`` to use the active cell scalars of the
+            :class:`pyvista.RectilinearGrid` to be either 5 or 12 to
+            determining the number of tetrahedra to generate per cell.
+
+            When a sequence, uses these value to subdivide the cells. When a
+            string, uses cell array rather than the active array to determine
+            the number of tetrahedra to generate per cell.
```

Comment:
```suggestion
            When set, subdivides some cells into 5 and some cells into 12. Set
            to ``True`` to use the active cell scalars of the
            :class:`pyvista.RectilinearGrid` to be either 5 or 12 to
            determining the number of tetrahedra to generate per cell.

            When a sequence, uses these values to subdivide the cells. When a
            string uses a cell array rather than the active array to determine
            the number of tetrahedra to generate per cell.
```